src/lib/typeUtils.ts

```diff
@@ -1,7 +1,9 @@
 const EMPTY_OBJ_REF = Object.freeze({});
```

Comment:
can we utilize `EMPTY_OBJ` from constants

Reply:
yep, good call